execution/executor-benchmark/src/transaction_generator.rs

```diff
@@ -310,15 +292,15 @@ impl TransactionGenerator {
                         .index(0);
                 let seed_account = &self.seed_accounts_cache[seed_index];
                 let txn = create_transaction(
-                    &seed_account.private_key,
-                    seed_account.public_key.clone(),
+                    seed_account.private_key(),
```

Comment:
I think we can replace this `create_transaction` with `sign_with_transaction_builder` from the local account